background/handle-unsupported-version.js

```diff
@@ -0,0 +1,18 @@
+function handleUnsupported() {
+  const checkIfUnsupported = () => {
+    const getVersion = () => {
+      let userAgent = /(Firefox|Chrome)\/([0-9.]+)/.exec(navigator.userAgent);
+      if (!userAgent) return { browser: null, version: null };
+      return { browser: userAgent[1], version: userAgent[2].split(".")[0] };
+    };
+
+    let { browser, version } = getVersion();
+    return (browser === "Chrome" && version < 81) || (browser === "Firefox" && version < 74);
+  };
```

Comment:
To maintain consistency with other parts of code, use
```js
      if (typeof browser !== "undefined") 
``` 
for browser checking instead of ua

Reply:
no because the ua has a multipurpose, getting the browser AND the version

Reply:
No, we want to maintain consistency. PLUS, I'm not sure if this will work in other Chromium browsers.

Reply:
no wl said it will, also there is not other way to get the version (except ff)